BEIMA.Backend.FT/DeviceFT.cs

```diff
@@ -92,6 +92,12 @@ public async Task DeviceNotInDatabase_AddDevice_CreatesNewDevice()
             Assert.That(getDevice.Location?.Notes, Is.EqualTo(device.Location.Notes));
             Assert.That(getDevice.Location?.Longitude, Is.EqualTo(device.Location.Longitude));
             Assert.That(getDevice.Location?.Latitude, Is.EqualTo(device.Location.Latitude));
+
+            // Cleanup
+            if(getDevice.Id != null)
```

Comment:
Cleanup is already done in the setup method, so that if you run a test, you can still look at what the state of the database is at the end of the test.